flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/debezium/reader/BinlogSplitReaderTest.java

```diff
@@ -324,7 +334,95 @@ public void testReadBinlogFromLatestOffset() throws Exception {
                 };
         List<String> actual =
                 readBinlogSplitsFromLatestOffset(dataType, configuration, expected.length);
-        assertEquals(Arrays.stream(expected).sorted().collect(Collectors.toList()), actual);
+        assertThat(actual, containsInAnyOrder(expected));
+    }
+
+    @Test
+    public void testReadBinlogCrossTransaction() throws Exception {
```

Comment:
Personally, I think this case doesn't cover processes, including: snapshot, restore from snapshot. Would be better to test on `MySqlSourceReader`. You can see `org.apache.flink.connector.testutils.source.reader.SourceReaderTestBase#testSnapshot` about how to test it.